test/e2e/density.go

```diff
@@ -250,163 +249,8 @@ var _ = Describe("Density", func() {
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
 
-			if itArg.runLatencyTest {
```

Comment:
I'm actually going to retain these since we still want to be able to run the latency test inline during the density test after the cluster has been primed.